metrics/write.go

```diff
@@ -128,14 +128,14 @@ func (c *Client) write(ctx context.Context) error {
 	log.Printf("Sending: %v timeseries, %v samples, %v timeseries per request, %v delay between requests\n", len(tss), c.config.RequestCount, c.config.BatchSize, c.config.RequestInterval)
 	ticker := time.NewTicker(c.config.RequestInterval)
 	defer ticker.Stop()
-	for ii := 0; ii < c.config.RequestCount; ii++ {
+	for ii := 0; c.config.RequestCount == -1 || ii < c.config.RequestCount; ii++ {
```

Comment:
I would suggest we remove this `for <"i" sheningans>`.  Main reason is that it's a bit not idiomatic  way of doing this (not popular in Go), also because (in theory) you can overflow `ii` integer.

```suggestion
	left := c.config.RequestCount // left equal to -1 means infinite amount of requests.
	for left != 0 {
	      left--
```